Answers.md

```diff
@@ -0,0 +1,11 @@
+1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+
+`Describe() is used to break down your test suite into components, whereas it() is where your individual tests are performed. You would use a describe() to indicate that that section of the test suite is going to be testing a certain endpoint. Inside the describe(), you would have one or more it(), depending on how many tests you need to do for that section of the test suite.`
+
```

Comment:
Good answers Jesse, I would add that `describe()` blocks are a great way to group _related_ tests together 👍